module/Finna/src/Finna/View/Helper/Root/Record.php

```diff
@@ -1277,4 +1277,52 @@ public function getOrganisationMenuPosition()
         return ('SolrAuth' === $source || $this->hasLargeImageLayout())
             ? 'inline' : 'sidebar';
     }
+
+    /**
+     * If deduplication selections are allowed to fetch holdings
```

Comment:
`Whether to load holdings for the default source on page load for deduplicated records.`